terraform/files/bin/openstack-kube-versions.inc

```diff
@@ -3,12 +3,12 @@
 # (c) Kurt Garloff <kurt@garloff.de>, 3/2022
 # SPDX-License-Identifier: Apache-2.0
 # Images from https://swift.services.a.regiocloud.tech/swift/v1/AUTH_b182637428444b9aa302bb8d5a5a418c/openstack-k8s-capi-images
-k8s_versions=("v1.18.20" "v1.19.16" "v1.20.15" "v1.21.14" "v1.22.17" "v1.23.16" "v1.24.15" "v1.25.13" "v1.26.8" "v1.27.5" "v1.28.1")
+k8s_versions=("v1.20.15" "v1.21.14" "v1.22.17" "v1.23.16" "v1.24.15" "v1.25.15" "v1.26.10" "v1.27.7" "v1.28.3")
 # OCCM, CCM-RBAC, Cinder CSI, Cinder-Snapshot (TODO: Manila CSI)
-occm_versions=(""         ""       "v1.21.1"   "v1.21.1"  "v1.22.2"  "v1.23.4"  "v1.24.6" "v1.25.6" "v1.26.3" "v1.27.2" "v1.28.0")
-#ccmr_versions=(""         ""        ""         ""         "v1.22.2"  "v1.23.4"  "v1.24.6" "v1.25.6" "v1.26.3" "v1.27.2" "v1.28.0")
-ccmr_versions=(""         ""        "v1.22.2"  "v1.22.2"  "v1.22.2"  "v1.23.4"  "v1.24.6" "v1.25.6" "v1.26.3" "v1.27.2" "v1.28.0")
-ccsi_versions=(""         ""        "v1.20.5"  "v1.21.1"  "v1.22.2"  "v1.23.4"  "v1.24.6" "v1.25.6" "v1.26.3" "v1.27.2" "v1.28.0")
+occm_versions=("v1.21.1"   "v1.21.1"  "v1.22.2"  "v1.23.4"  "v1.24.6" "v1.25.6" "v1.26.4" "v1.27.3" "v1.28.1")
+#ccmr_versions=(""         ""         "v1.22.2"  "v1.23.4"  "v1.24.6" "v1.25.6" "v1.26.4" "v1.27.3" "v1.28.1")
+ccmr_versions=("v1.22.2"  "v1.22.2"  "v1.22.2"  "v1.23.4"  "v1.24.6" "v1.25.6" "v1.26.4" "v1.27.3" "v1.28.1")
+ccsi_versions=("v1.20.5"  "v1.21.1"  "v1.22.2"  "v1.23.4"  "v1.24.6" "v1.25.6" "v1.26.4" "v1.27.3" "v1.28.1")
 min_snapshot_master="v1.21.0"
```

Comment:
I think we can safely drop v1.20 because it ends life 2022-02-28 (it will be more than 2 years old in our upcoming release) and also there are no images for this version.

Then we can probably remove `min_snapshot_master` logic too.